metadata.json

```diff
@@ -102,12 +103,8 @@
   ],
   "requirements": [
     {
-      "name": "pe",
-      "version_requirement": ">= 3.7.0 < 2015.4.0"
-    },
-    {
       "name": "puppet",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 4.5.0 < 5.0.0"
```

Comment:
@DavidS is there a specific Puppet feature in use that scopes the requirement to >=4.5.0? This restricts use to PE 2016.2+ which is ok but I'd like to know what's gained in this module as a trade off.